docs/source/actor-rhel7-to-rhel8.md

```diff
@@ -161,20 +163,40 @@ class MyNewActor(Actor):
 
 Final report is generated in **"txt"** and **"json"** format in `/var/log/leapp` directory at the end of either leapp preupgrade or leapp upgrade execution.
 
-### Reporting tips
-
-Apart from the above example you can also suggest a remediation.
+### Reporting tips and good practices
+
+**Same type reports share the same title**
+
+To make your reports easier to read and aggregate please use same report title for checks that have the same semantics.
+For example, if your actor is checking for deprecated modules and creating a report entry each time one is found,
+the way to represent it in the report will be
+
+```diff
+create_report([
+    reporting.Title(
+-        'Upgrade process was interrupted because a deprecated module {0} is enabled'.format(module),
++        'Upgrade process was interrupted because a deprecated module is enabled',
+    reporting.Summary(
+        'Module {0} was surpassed by shiny-new-{0} and therefore it was '
+        'removed from RHEL-8. Keeping it in the configuration may '
+        'lock out the system thus it is necessary to disable it '
+        'before the upgrade process can continue.'.format(module)
+    ),
```

Comment:
I would go even further and skip generating of multiple reports for the same thing So summary would looks like:
```diff
-        'Module {0} was surpassed by shiny-new-{0} and therefore it was '
-        'removed from RHEL-8. Keeping it in the configuration may '
-        'lock out the system thus it is necessary to disable it '
-        'before the upgrade process can continue.'.format(module)
+        'Modules below have been surpassed by new modules and therefore these '
          ....
+        'before the upgrade process can continue. The list of affected modules:'
+        '    - modulA  -> Amod'
+        '    - modulB  -> Bmod'
```

Reply:
or it could start as well as 'We detected deprecated modules that are replaced....'

Reply:
I'd say we shouldn't restrict 1-type-1-report-entry-per-actor (imagine we have a remediation for each module, some are hints, some are commands, but they must be run in specific order), but I'll add your example as another way of fixing the trouble.

Reply:
@fernflower But in this very specific case, it's just about names of modules. everythig else is same. I think we could later create list of examples, documenting several cases and explain why we choose to report them in this of another way.